src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import cn from 'classnames';
+
+import { Filter } from '../types/Filter';
+
+const FILTER_LINKS: Filter[] = ['All', 'Active', 'Completed'];
```

Comment:
Don't need to create additional array, use `Object.value(your created enum)` for creating array